npm run test fails for conflict with ~/.tldrrc

## Expected behaviour
`npm run test` should pass

## Actual behaviour
It doesn't pass for the current version of https://github.com/tldr-pages/tldr-node-client/commit/ef79714ef1cd50a45894059b26573fc3815afaac

Specifically the render.spec.js tests won't pass saying `TypeError: Cannot read property 'commandName' of undefined`. 

## Log, debug output

```
$ npm run test

> tldr@3.2.3 test /home/peng/github/tldr-node-client
> mocha test --require=env-test



  Cache
    ✓ should return a positive number on lastUpdate (1365ms)
    update()
      ✓ should use randomly created temp folder
      ✓ should remove temp folder after cache gets updated
    getPage()
      ✓ should return page contents for ls
      ✓ should return empty contents for svcs on OSX
      ✓ should return page contents for svcs on SunOS
      ✓ should return empty contents for non-existing page

  Config
    ✓ should load the default config
    ✓ should override the defaults with content from .tldrrc
    ✓ should validate the custom config format

  Index building
    failure
      ✓ shortIndex should not be created
    success
      ✓ correct shortIndex should be created

  Index
    ✓ should return correct list of all pages
    ✓ should return correct short index on getShortIndex()
    findPlatform()
      ✓ should find Linux platform for dd command
      ✓ should find platform common for cp command
      ✓ should not find platform for svcs command on Linux
      ✓ should not find platform for non-existing command
    commandsFor()
      ✓ should return correct list of pages for Linux
      ✓ should return correct list of pages for OSX
      ✓ should return correct list of pages for SunOS

  Parser
    ✓ parses the command name
    ✓ parses the description
    ✓ can parse the description on multiple lines
    ✓ does not escape HTML entities
    ✓ parses example descriptions and codes
    ✓ does not escape HTML in the examples either
    ✓ parses all the examples
    ✓ leaves out malformed examples
    ✓ should parse description with inline code
    ✓ should parse examples with inline code
    ✓ should parse code examples with unix redirects ">", "<", ">>" and "<<<"
    See also section
      ✓ should parse seeAlso commands when mentioned in description
      ✓ should parse seeAlso commands when mentioned in examples
      ✓ should have only unique seeAlso commands when mentioned a few times

  Platform
    getPreferredPlatform
      ✓ should return the running platform with no configuration
      ✓ should overwrite the running platform if configured
      ✓ should return current system platform if configuration is wrong
    isSupported
      ✓ should tell that Linux, OSX, SunOS and Win32 are supported
    resolve
      ✓ should resolve tar command with specific OS and common folder

  Render
    1) surrounds the output with blank lines
    2) contains the command name
    3) contains the description name
    4) highlights replaceable {{tokens}}
    5) should correctly render see also section

  Search
    ✓ should create index
    ✓ should perform searches

  Theme
    Rendering
      ✓ should render name with green and bold
      ✓ should render description with red and underline
      ✓ should render example description with blue
      ✓ should render example code with blue
      ✓ should render example argument with yellow, dim, underline
    Rendering with new theme colors
      ✓ should render name with greenBright and bold
      ✓ should render description with greenBright and bold
      ✓ should render example description with greenBright
      ✓ should render example code with redBright
      ✓ should render example argument with white


  52 passing (1s)
  5 failing

  1) Render
       surrounds the output with blank lines:
     TypeError: Cannot read property 'commandName' of undefined
      at Theme.getStylingFunction (lib/theme.js:25:28)
      at Theme.renderCommandName (lib/theme.js:30:17)
      at Object.exports.toANSI (lib/render.js:28:28)
      at Context.it (test/render.spec.js:14:23)

  2) Render
       contains the command name:
     TypeError: Cannot read property 'commandName' of undefined
      at Theme.getStylingFunction (lib/theme.js:25:28)
      at Theme.renderCommandName (lib/theme.js:30:17)
      at Object.exports.toANSI (lib/render.js:28:28)
      at Context.it (test/render.spec.js:24:23)

  3) Render
       contains the description name:
     TypeError: Cannot read property 'commandName' of undefined
      at Theme.getStylingFunction (lib/theme.js:25:28)
      at Theme.renderCommandName (lib/theme.js:30:17)
      at Object.exports.toANSI (lib/render.js:28:28)
      at Context.it (test/render.spec.js:34:23)

  4) Render
       highlights replaceable {{tokens}}:
     TypeError: Cannot read property 'commandName' of undefined
      at Theme.getStylingFunction (lib/theme.js:25:28)
      at Theme.renderCommandName (lib/theme.js:30:17)
      at Object.exports.toANSI (lib/render.js:28:28)
      at Context.it (test/render.spec.js:44:23)

  5) Render
       should correctly render see also section:
     TypeError: Cannot read property 'commandName' of undefined
      at Theme.getStylingFunction (lib/theme.js:25:28)
      at Theme.renderCommandName (lib/theme.js:30:17)
      at Object.exports.toANSI (lib/render.js:28:28)
      at Context.it (test/render.spec.js:58:23)



npm ERR! code ELIFECYCLE
npm ERR! errno 5
npm ERR! tldr@3.2.3 test: `mocha test --require=env-test`
npm ERR! Exit status 5
npm ERR! 
npm ERR! Failed at the tldr@3.2.3 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/peng/.npm/_logs/2018-07-23T21_08_54_098Z-debug.log

```

## Environment
- Operating system: Ubuntu 16.04.3 LTS
- NodeJS version: v10.6.0

--- a/test/render.spec.js
+++ b/test/render.spec.js
@@ -2,12 +2,27 @@
 
 const render = require('../lib/render');
 const config = require('../lib/config');
+const sinon = require('sinon');
 
 describe('Render', () => {
 
   beforeEach(() => {
-    config.reset();
-    config.get().theme = 'base16';
+    sinon.stub(config, 'get').returns({
+      'themes': {
+        'base16': {
+          'commandName': 'bold',
+          'mainDescription': '',
+          'exampleDescription': 'green',
+          'exampleCode': 'red',
+          'exampleToken': 'cyan'
+        }
+      },
+      'theme': 'base16'
+    });
+  });
+
+  afterEach(() => {
+    config.get.restore();
   });
 
   it('surrounds the output with blank lines', () => {

